tests/structure_test.js

```diff
@@ -849,6 +849,26 @@ describe('structure', function () {
         ref.cursor().update(function () { return 'updated'; });
       });
 
+      it('should trigger change listener for depending observed references', function (done) {
+        var structure = new Structure({
+          data: {a: {b: {c: "value"}}}
+        });
+
+        var ref = structure.reference(['a', 'b']);
+        ref.observe(function () { done(); });
+        structure.cursor('a').update(function (old) { return old.setIn(['b', 'c'], 'updated'); });
+      });
+
+      it('should trigger change listener for depending observed references', function (done) {
+        var structure = new Structure({
+          data: {a: {b: {c: "value"}}}
+        });
+
+        var ref = structure.reference(['a', 'b']);
+        ref.observe(function () { done(); });
+        structure.cursor().update(function (old) { return old.setIn(['a', 'b', 'c'], 'updated'); });
```

Comment:
Basically no, I just wanted to be sure that root is handled the same way as subroot